src/Microsoft.Extensions.Configuration.Abstractions/IConfiguration.cs

```diff
@@ -36,5 +36,6 @@ public interface IConfiguration
         IEnumerable<IConfigurationSection> GetChildren();
 
         IChangeToken GetReloadToken();
+        IConfigurationRoot Root { get; }
```

Comment:
Alternatively we can just nuke it entirely since we are exposing root now, and it was just directing to the root method, we can remove this off IConfiguration and leave it only on IConfigurationRoot next to Reload()...even better?